src/hidpp20.h

```diff
@@ -117,6 +117,13 @@ enum hidpp20_battery_status {
 	BATTERY_STATUS_INVALID,
 };
 
+enum hidpp20_led_mode {
+	HIDPP20_LED_OFF = 0,
+	HIDPP20_LED_ON = 1,
+	HIDPP20_LED_CYCLE = 03,
```

Comment:
03 in octal notation? That seems odd. Either decimal or switch all to hex please